docs/hub/repositories-licenses.md

```diff
@@ -84,6 +84,7 @@ A full list of the available licenses is available here:
 | Llama 3.2 Community License Agreement                          | `llama3.2`                               | <!-- license: https://huggingface.co/meta-llama/Llama-3.2-1B/blob/main/LICENSE.txt -->
 | Llama 3.3 Community License Agreement                          | `llama3.3`                               | <!-- license: https://huggingface.co/meta-llama/Llama-3.3-70B-Instruct/blob/main/LICENSE -->
 | Llama 4 Community License Agreement                            | `llama4`                                 | <!-- license: https://huggingface.co/meta-llama/Llama-4-Scout-17B-16E-Instruct/blob/main/LICENSE -->
+| FAIR Noncommercial Research License                            | `fair-noncommercial-research-license`    | <!-- license: https://huggingface.co/facebook/fair-noncommercial-research-license/blob/main/LICENSE -->                                     |
```

Comment:
Can we move this up in the list (the licenses are in alphabetical order) 🙏 - 

minor nit: also there's an extra space in the last line